python/benchmark_filesystem.py

```diff
@@ -14,7 +14,6 @@ def benchmark(
     crop_and_resize: bool = typer.Option(
         True, help="Crop and resize the images on the fly"
     ),
-    concurrency: int = typer.Option(64, help="The number of coroutines"),
```

Comment:
the whole point of this PR: remove this line :D we don't care anymore, it will adapt